doc/source/reference/routines.linalg.rst

```diff
@@ -18,6 +18,18 @@ or specify the processor architecture.
 .. _OpenBLAS: https://www.openblas.net/
 .. _threadpoolctl: https://github.com/joblib/threadpoolctl
 
+The SciPy library also contains a ``linalg`` submodule, and there is overlap
```

Comment:
Might be handy to link this:
```suggestion
The SciPy library also contains a `~scipy.linalg` submodule, and there is overlap
```

Reply:
The module `scipy.linalg` is referenced two sentences down, but it makes sense have it here in the first sentence.  What does the `~` do?  I didn't use it below, and the link appears to be generated correctly in the HTML output.

Reply:
`~` omits the `scipy.` prefix from the link text

Reply:
Fixed in the latest updated.